components/dashboard/src/projects/new-project/NewProjectRepoList.tsx

```diff
@@ -39,17 +38,9 @@ export const NewProjectRepoList: FC<Props> = ({ filteredRepos, noReposAvailable,
                             </div>
                             <div className="flex justify-end">
                                 <div className="h-full my-auto flex self-center opacity-0 group-hover:opacity-100 items-center mr-2 text-right">
-                                    {!r.inUse ? (
-                                        <Button onClick={() => onRepoSelected(r)} loading={isCreating}>
-                                            Select
-                                        </Button>
-                                    ) : (
-                                        <p className="text-gray-500">
-                                            Project already
-                                            <br />
-                                            exists.
-                                        </p>
-                                    )}
+                                    <Button onClick={() => onRepoSelected(r)} loading={isCreating}>
+                                        Select
```

Comment:
**issue**: When you import a repository, clicking on Enable Prebuilds  button within a few seconds goes into a redirect loop and stops. Could be unrelated to the changes here. 🤷 

<img width="1512" alt="Screenshot 2023-09-22 at 19 53 27" src="https://github.com/gitpod-io/gitpod/assets/120486/f693ace8-1c04-4a6d-a48e-dccef602856b">

Reply:
I noticed that too, it does a link to the settings page (doesn't navigate in client), but then the page partially loads and flashes, then seems to get hung up.